app/javascript/stylesheets/components/button.scss

```diff
@@ -0,0 +1,29 @@
+.govuk-button {
```

Comment:
Should discuss a better way of doing this I think. Quick and dirty for now..
Don't like using `!important` to override but it is needed to override the govuk style